doc/api/deprecations.md

```diff
@@ -717,24 +717,6 @@ Type: Documentation-only
 The [`ecdh.setPublicKey()`][] method is now deprecated as its inclusion in the
 API is not useful.
 
-### DEP0032: `node:domain` module
-
-<!-- YAML
-changes:
-  - version:
-    - v6.12.0
-    - v4.8.6
-    pr-url: https://github.com/nodejs/node/pull/10116
-    description: A deprecation code has been assigned.
-  - version: v1.4.2
-    pr-url: https://github.com/nodejs/node/pull/943
-    description: Documentation-only deprecation.
--->
-
-Type: Documentation-only
```

Comment:
We need to first runtime deprecate it.

Reply:
Note my "draft PR to check citgm" remark (which is inconclusive so far - citgm seems to be in bad shape)

That said... the domain module has basically always been deprecated<sup>0</sup>. Standard procedures are nice but sometimes it makes no sense to follow them. 

<sup>0</sup> it was added in v0.10.0 and deprecated in v1.0.0. We'll not talk about the brown paper bag releases that were v0.12.x.